src/fosslight_dependency/package_manager/Go.py

```
@@ -71,9 +71,12 @@ def run_plugin(self):
         self.append_input_package_list_file(self.tmp_file_name)
 
         cmd_tree = "go mod graph"
-        ret_cmd_tree = subprocess.check_output(cmd_tree, shell=True, text=True, encoding='utf-8')
-        if ret_cmd_tree != 0:
-            self.parse_dependency_tree(ret_cmd_tree)
+        try:
+            ret_cmd_tree = subprocess.check_output(cmd_tree, shell=True, text=True, encoding='utf-8')
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)